cmd/deploy/service-set.go

```diff
@@ -61,8 +63,9 @@ func executeDeployServiceSet(cmd *cobra.Command) {
 		deployServiceSetRequest.Name = serviceSetName
 	}
 
-	err := serviceClient.DeployServiceSet(&ctx, &deployServiceSetRequest)
+	err := serviceClient.DeployServiceSet(&ctx, &deployServiceSetRequest, traceID)
 	if err != nil {
-		log.Fatal("Failed to deploy service ", err)
+		log.Info("TraceId: ", traceID)
```

Comment:
We want to display traceId to the users in case of failure.
If logged as debug message, user will then have to run CLI in debug mode explicitly to see traceId.

Reply:
The purpose of trace id is to track all logs for a particular request, it should not be limited to just errors. In case of errors we can log the trace id in the error message itself

Reply:
Correct, trace is associated with all the requests but shown to the user only in case of errors

Reply:
Yes so in case of errors we can log it again with error log level